vts/plugins/scheme-cca-ssd-platform/main.go

```diff
@@ -375,11 +376,11 @@ func matchPlatformConfig(evidence psatoken.IClaims, endorsements []Endorsements)
 		return false
 	}
 	if len(endorsements) > 1 {
-		log.Printf("matchPlatformConfig failed number of cca config %d > 1 ", len(endorsements))
+		log.Debug("matchPlatformConfig failed number of cca config %d > 1 ", len(endorsements))
```

Comment:
I am unable to understand what we want to say with this message. Is it:
```golang
    log.Debug("got %d CCA configuration endorsements, want 1", len(endorsements))
```
If so, we seem to be constraining the number of possible configurations to one only.  Is this intended?  Shouldn't we cycle over all the existing until we find a match instead?